tedana/workflows/tedana.py

```diff
@@ -159,8 +157,12 @@ def tedana(data, tes, mixm=None, ctab=None, manacc=None, strict=False,
         n_components, dd = decomposition.tedpca(catd, OCcatd, combmode, mask, t2s, t2sG,
                                                 stabilize, ref_img,
                                                 tes=tes, kdaw=kdaw, rdaw=rdaw, ste=ste)
+<<<<<<< HEAD
+        mmix_orig = decomposition.tedica(n_components, dd, conv, fixed_seed, cost=cost)
+=======
         mmix_orig = decomposition.tedica(n_components, dd, conv, fixed_seed, cost=initcost,
                                          final_cost=finalcost, verbose=debug)
+>>>>>>> upstream/master
```

Comment:
I think this might be a mistake from when you resolved merge conflicts.

Reply:
Definitely. I'm about to push a fix for this and several other syntax errors ! Thanks for the heads up 👍